### PR TITLE
Compliance and anti trust statement

### LINKED ISSUE #6: Compliance and Antitrust Statement of MA
Companies today expect organizations as MA to have a Compliance and Antitrust statement
Example ISO: https://www.iso.org/files/live/sites/isoorg/files/developing_standards/docs/en/competition_law_guidelines.pdf

As proposed by the FMI Steering committee and agreed with @MartinOtter , we will use a branch of this repo to work on a proposal.
It is not intended to propose Bylaw changes; it will be  sufficient to have a vote by the MA Assembly to accept a proposed text.

I started with https://github.com/modelica/MA-Bylaws/tree/ComplianceAndAntiTrustStatement in a branch, feel free to comment and improve the text.

@hubertus65 and @chrbertsch will work on the proposal; anyone is free to contribute.

### PR DESCRIPTION
**Pull request by chrbertsch**
Resolves #6 

As discussed at the annual Modelica meeting, this is a PR to discuss the proposal for a Compliance and AntiTrust statement of MA with the larger group.

### CODE REVIEW

File: MA-ComplianceAndAntitrustStatement.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Compliance and Antitrust Guidelines of the Modelica Association (Version 1.0 - draft)
+
+The purpose of these Compliance and Antitrust Guidelines (“Guidelines”)  is  to raise awareness of, and provide broad guidance on, competition law issues to you as a participant in the Modelica Association (“MA”) standards development process.
```
Comment (beutlich) [inline MA-ComplianceAndAntitrustStatement.md:3]: Why extra space before and after "is"?

Comment (MartinOtter) [reply MA-ComplianceAndAntitrustStatement.md:3]: The extra spaces should be just removed

Comment (beutlich) [inline MA-ComplianceAndAntitrustStatement.md:3]:
```suggestion
The purpose of these Compliance and Antitrust Guidelines (“Guidelines”) is to raise awareness of, and provide broad guidance on, competition law issues to you as a participant in the Modelica Association (“MA”) standards development process.
```

**Comment by chrbertsch** [comment]
There have been no futher activities for 1/2 year. Can we merge this?  (Or do we need a formal approval process?)

**Comment by HansOlsson** [comment]
> There have been no futher activities for 1/2 year. Can we merge this? (Or do we need a formal approval process?)

One minor issue I realized is that it is partially copied from a public document that seems to be copyright by ISO. Is there any problem with that (especially since we are not part of ISO)?

**Comment by chrbertsch** [comment]
> One minor issue I realized is that it is partially copied from a public document that seems to be copyright by ISO. Is there any problem with that (especially since we are not part of ISO)?

Shall we cite ISO? https://www.iso.org/files/live/sites/isoorg/files/developing_standards/docs/en/competition_law_guidelines.pdf 
And perhaps simply ask them? @MartinOtter or @MABackoffice : could you plesae do this?

**Comment by HansOlsson** [comment]
> > One minor issue I realized is that it is partially copied from a public document that seems to be copyright by ISO. Is there any problem with that (especially since we are not part of ISO)?
> 
> Shall we cite ISO? https://www.iso.org/files/live/sites/isoorg/files/developing_standards/docs/en/competition_law_guidelines.pdf

Citing instead of copying seem safe from a copyright perspective; but has the added cost that the link may become stale and that we need to ensure that we are of any changes in the contents.

> And perhaps simply ask them? @MartinOtter or @MABackoffice : could you plesae do this?

**Comment by chrbertsch** [comment]
@MABackoffice : is there any progress on this?

**Comment by andreas-junghanns** [comment]
@chrbertsch : Do we need to put this on the agenda for tomorrow or next week´s MA Assembly (again) to finally move this forward?

**Comment by chrbertsch** [comment]
> @chrbertsch : Do we need to put this on the agenda for tomorrow or next week´s MA Assembly (again) to finally move this forward?

The content does not seem to be controversial, the remaining issue is, if we have have copied too much from ISO to violate their copyright. 
@MABackoffice : Could you please check with ISO, of they care?

**Comment by chrbertsch** [comment]
@MABackoffice : could you please clarify with ISO? Or @hubertus65 : do you have contacts there?

**Comment by MABackoffice** [comment]
ISO said "We are happy to grant you the permission to use the content from that publication provided that you reference the source. "

**Comment by chrbertsch** [comment]
> ISO said "We are happy to grant you the permission to use the content from that publication provided that you reference the source. "

As we already cite ISO, what is missing to publish this?

**Comment by MABackoffice** [comment]
> > ISO said "We are happy to grant you the permission to use the content from that publication provided that you reference the source. "
> 
> As we already cite ISO, what is missing to publish this?

I think they were just emphasizing that we need to cite them. Their email should suffice as approval to use their statement.

**Pull request merged**